lego/apps/surveys/fixtures/survey_template_fixtures.yaml

```diff
@@ -54,6 +54,13 @@
     event: 8
     template_type: 'event'
 
+- model: surveys.Survey
+  pk: 9
+  fields:
+    title: 'Template: alternate'
+    event: 9
+    template_type: 'alternate'
```

Comment:
I think it might be `alternative_presentation`, but I might be wrong.

Reply:
Yes